zproject/urls.py

```diff
@@ -400,6 +400,8 @@
     # Used for public-only realm exporting
     url(r'^export/realm$', rest_dispatch,
         {'POST': 'zerver.views.public_export.public_only_realm_export'}),
+    url(r'^public_exports$', rest_dispatch,
+        {'GET': 'zerver.views.public_export.get_public_exports'}),
```

Comment:
This endpoint should be `GET /export/realm`; we'll ultimately want to do `DELETE /export/realm/15` as well for the deletion request.

Reply:
Updated `GET`, holding off on `DELETE` pending further feedback.